app/components/Footer/style.module.css

```diff
@@ -0,0 +1,73 @@
+.footer {
+  background: var(--gradiente-light);
+  word-break: break-word;
+  padding-top: 20rem;
+  margin-top: -16rem;
+  z-index: 0;
+  position: relative;
+}
+
+.infoContainer {
+  padding: 4rem 0;
+}
+
+.infoContainer,
+.infoContainer div,
+.infoContainer address {
+  display: flex;
+  flex-direction: column;
+  gap: 3rem;
+}
+
+.infoContainer div,
+.infoContainer address {
+  gap: 0.5rem;
+}
+
+.infoContainer a:hover,
+.infoContainer a:active {
+  text-decoration: underline;
+}
+
+.infoContainer a:visited {
+  text-decoration: none;
+}
+
+.infoSubtitle {
+  color: var(--neutral-gray-darkest);
+  text-wrap: wrap;
+}
+
+.infoField {
+  display: inline-flex;
+  gap: 0.5rem;
+
+  font-style: normal;
+  line-height: 22px;
```

Comment:
Não é uma boa ter line height em px. Se for usar um valor absoluto, vamos colocar rem. Eu tentaria usar em porcentagem, no formato 1.3, 1.25 etc

Reply:
Alterado para o valor de 1.375 para ficar 22px.